### PR TITLE
Update to new phylopic API

### LINKED ISSUE #23: dig into new API
http://www.phylopic.org.s3-website-us-west-2.amazonaws.com/api/docs/2.0/

emailed asking about API keys and base url

### LINKED ISSUE #35: Help wanted moving to new Phylopic API
see issue #23 

Anyone that's contributed to this package or using it want to help move to the new API? Seems there's a bit of a time crunch, and I don't really have time right now. 

❤️

### LINKED ISSUE #36: Error loading image data
<details> <summary><strong>─ Session info 
 setting  value
 version  R version 4.2.1 (2022-06-23)
 os       macOS Monterey 12.6.2
 system   x86_64, darwin17.0
 ui       RStudio
 language (EN)
 collate  en_US.UTF-8
 ctype    en_US.UTF-8
 tz       America/New_York
 date     2023-02-24
 rstudio  2022.12.0+353 Elsbeth Geranium (desktop)
 pandoc   NA</strong></summary>

</details>

I recently used rphylopic to add images to ggplot figures for a publication, I want to make some changes to these figures for my dissertation but the code I used to save the image data for the phylopic silhouettes is giving me an error. This code worked before and I checked that the phylopic image I am referencing still exists.  An example below

```r
library(rphylopic)
lynx_phylo <- image_data("0957f561-d68e-499b-b1ad-20f444064033", size = 256)[[1]]
```

The error returned is:
Error in png::readPNG(res$content) : libpng error: Not a PNG file

I found a similar issue online <a href = "https://github.com/palaeoverse-community/rphylopic/issues/22" target = "_blank">[libpng errors #22]</a> but the solution code did not work for me, example below. 

```r
id <- '0957f561-d68e-499b-b1ad-20f444064033'
z <- name_images(uuid = id)
```
Which returned this error: 
Error: Not Found (HTTP 404)

Any assistance resolving this would be greatly appreciated.

### PR DESCRIPTION
**Pull request by willgearty**
This includes all of the changes related to reworking the package for the new phylopic API. It is still a work-in-progress.

Fixes #35. Fixes #23. Fixes #36.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
+    person(given = "Scott", family = "Chamberlain", email = "myrmecocystus@gmail.com",
+      role = c("aut"), comment = c(ORCID = "0000-0003-1444-9135")),
+    person(given = "David", family = "Miller", email = "dave@ninepointeightone.net",
+      role = c("ctb"))
   )
-License: MIT + file LICENSE
-URL: https://github.com/sckott/rphylopic
-BugReports: https://github.com/sckott/rphylopic/issues
-Roxygen: list(markdown = TRUE)
+License: GPL (>= 3)
```
Comment (sckott) [inline DESCRIPTION:18]: curious why the change in license?

Comment (LewisAJones) [reply DESCRIPTION:18]: Hey @sckott, this is the license we typically use for all our code (e.g. [palaeoverse](https://github.com/palaeoverse-community/palaeoverse)). As we've overhauled a lot of the package, we were planning to release it as a new major version, and it seemed an appropriate time to update the license to bring it in line with the other packages we maintain. Happy to discuss more if there is an issue?
